StepsToCare/CareEntry.ascx.cs

```diff
@@ -49,9 +49,10 @@ namespace RockWeb.Plugins.rocks_kfs.StepsToCare
         DefaultBooleanValue = false,
         Key = AttributeKey.AllowNewPerson )]
 
-    [GroupRoleField( null, "Group Type and Role",
-        Description = "Select the group Type and Role of the leader you would like auto assigned to care need. If none are selected it will not auto assign the small group member to the need. ",
+    [CustomEnhancedListField( "Group Type Roles",
+        Description = "Select the Group Type Roles of the leaders you would like auto assigned to care need when the Person is a member of this type of group. If none are selected it will not auto assign the small group member with the appropriate role to the need. ",
```

Comment:
```suggestion
        Description = "Select the Group Type > Roles for the group members you would like auto assigned to Care Needs created for people who are in groups of these types. If none are selected it will not auto assign the group member with the appropriate role to the need. ",
```

Reply:
Might need to change in migration and job setting on the assembly side.